Userland/Libraries/LibWeb/StorageAPI/NavigatorStorage.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/Forward.h>
```

Comment:
When I pulled this branch locally, clangd complained loudly (rightly) about missing includes. You will want:
```c++
#include <LibJS/Forward.h>
#include <LibJS/Heap/GCPtr.h>
#include <LibWeb/Forward.h>
```

And also `#include <LibWeb/Bindings/PlatformObject.h>` with the above virtual getter declared.

Reply:
Thank you! Done :)